### PR TITLE
add : badge component

### PR DESCRIPTION
**Pull request by Varun-28**
Done with badge component, It is two types : 
👉Profile badge
👉Icon badge
badge1
badge2

### CODE REVIEW

**Review by ZuberDunge — COMMENTED** [review]
looks good to me/

File: css/component_styles/badge.css [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+.dnd {
+    background-color: var(--color-error);
+}
+
+/* number badge */
+.shopping-cart {
+    font-size: 2rem;
+}
+
+.badge-ico {
```
Comment (prachi-sahani) [inline css/component_styles/badge.css:32]: badge-icon?

**Comment by hinanshisuthar** [comment]
You can merge. Looks good!

**Review by prachi-sahani — COMMENTED** [review]
looks good to me. 👍🏻 
Suggestion - put css for documentation in separate folder/file(as you have mentioned last two classes are for docs)

File: css/component_styles/badge.css (hunk 1 above)
Comment (Varun-28) [reply css/component_styles/badge.css:32]: Yaa ! 😅, actually there were two classes one badge-icon for the entire badge with icon and this badge-ico class for the badge on the icon

**Pull request merged**